acceptance/analyzer_test.go

```diff
@@ -820,6 +789,51 @@ func testAnalyzerFunc(platformAPI string) func(t *testing.T, when spec.G, it spe
 				})
 			})
 
+			when("called with previous image", func() {
+				it.Before(func() {
+					h.SkipIf(t, api.MustParse(platformAPI).Compare(api.MustParse("0.7")) < 0, "Platform API < 0.7 does not support -previous-image")
+				})
+
+				when("auth registry", func() {
+					var authRegAppImage, appAuthConfig string
+
+					it.Before(func() {
+						metadata := minifyMetadata(t, filepath.Join("testdata", "analyzer", "app_image_metadata.json"), platform.LayersMetadata{})
+						authRegAppImage, appAuthConfig = buildAuthRegistryImage(
+							t,
+							"some-app-image-"+h.RandString(10),
+							filepath.Join("testdata", "analyzer", "app-image"),
+							"--build-arg", "fromImage="+containerBaseImage,
+							"--build-arg", "metadata="+metadata,
+						)
+					})
+
+					it("writes analyzed.toml with previous image identifier", func() {
+						execArgs := []string{
+							ctrPath(analyzerPath),
+							"-previous-image", authRegAppImage,
+							"some-fake-image",
+						}
```

Comment:
As an aside, I love this. It makes it so much easier to understand how we're exec'ing the analyzer (what we care most about) instead of having to read all of the call to `DockerRunAndCopy`.